packages/eslint/index.js

```diff
@@ -47,7 +47,6 @@ module.exports = {
         ],
       },
     ],
-    'import/no-cycle': 'error',
```

Comment:
Why this was removed?

Reply:
To solve the following error:
![image](https://user-images.githubusercontent.com/39802870/103412017-f3398780-4b51-11eb-877e-9c9bf96c9133.png)
It is related to this issue https://github.com/typescript-eslint/typescript-eslint/issues/1333#issuecomment-566036856